examples/react_webpack/package.json

```diff
@@ -10,6 +10,7 @@
         "css-loader": "^3.2.0",
         "csstype": "^2.2.0",
         "http-server": "^0.11.1",
+        "protobufjs": "^6.8.8",
```

Comment:
this seems like a bug. the react_webpack example doesn't depend on protobufjs so it should come transitively from `@bazel/typescript`  or `@bazel/worker` I suspect?

Reply:
I never could figure this one out, it seemed like this should have come from a transitive dep, but yarn wasn't installing it, maybe because yarn doesn't understand deps on local packages properly?
I'll try removing it again to see if there is a different fix.

Reply:
okay waiting on you for next review pass then, ping me when ready